src/charts/c3/c3.directive.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.directive:c3Chart
+ *
+ * @description
+ *   Directive for wrapping c3 library
+ *
+ *   Note: The 'patternfly.charts' module is not a dependency in the default angular 'patternfly' module.
+ *   In order to use patternfly charts you must add 'patternfly.charts' as a dependency in your application.
+ *
+ *
+ * @param {string} id the ID iof the container that the chart should bind to
+ * @param {expression} config the c3 configuration options for the chart
+ *
+ * @example
+ <example module="patternfly.charts">
+<file name="index.html">
+  <div ng-controller="ChartCtrl">
+    <div c3-chart id="chartId"  config="chartConfig"></div>
+  </div>
+ </file>
+
+<file name="script.js">
+ function ChartCtrl($scope) {
+    $scope.chartConfig = {"donut":
+                     {"title":"MHz Used",
+                      "label":{"show":false},
+                      "width":10
+                     },
+                     "size":{"height":130},
+                     "legend":{"show":false},
+                     "color":{"pattern":["#CC0000","#D1D1D1"]},
+                     "tooltip":{},
+                     "data":{"columns":[["Used","950"],["Available",50]],
+                     "type":"donut",
+                     "donut":{
+                         "label":{"show":false}
+                      },
+                      "groups":[["used","available"]],
+                      "order":null
+                      }
+                  };
+ }
+ </file>
+ </example>
+ */
+(function(c3){
+  'use strict';
+
+  angular.module('patternfly.charts')
+  .directive('c3Chart', ['$timeout', function($timeout) {
```

Comment:
Are you saying the 'c3Chart' directive doesn't match the file name 'c3.directive.js?
Following the established naming patter already in repo, ie 'card.directive.sh' = 'pfCard'.,
So perhaps this directive should be 'pfC3Chart' to follow convention and differentiate between the c3 js lib.